### ISSUE TITLE
allow to filter "younger than" for construction year

### ISSUE THREAD

**Issue by ralfhauser** [mmmatthew/water-fountains.org#115]
as per issue #71 , it is only possible to filter by "older than" - it may be equally interesting to see the ones built recently / "after year x"

**Comment by mmmatthew** [comment]
should one be able to filter for a range, e.g. both older than x and younger than y simultaneously? Or should the date filter be either younger than OR older than?

**Comment by ralfhauser** [comment]
Please propose a step-up phasing with increasingly powerful/elaborate features

**Comment by mmmatthew** [comment]
I propose turning the "vor" into a dropdown where "nach" can also be selected.
![image](https://user-images.githubusercontent.com/10126798/53648748-f9523a80-3c40-11e9-9790-274901cd594d.png)

**Comment by ralfhauser** [comment]
1) drop-down is a great idea
2) "range" proposed to a later phase (or seperate RFE once this one is done)

**Comment by ralfhauser** [comment]
wonderful

**Comment by ralfhauser** [comment]
![image](https://user-images.githubusercontent.com/1681677/55776171-21cd2000-5a9c-11e9-85d5-6bfd706e0e60.png)
upon clicking into (3) or (4), automatically put the checkbox to the left

same with foto and water-type filter
